Pollination-apps/custom-chart.md

```diff
@@ -0,0 +1,53 @@
+# Add a custom chart to your Pollination app
+
+Install the following libraries first
```

Comment:
I think you should link from the first `chart` document to this one and vice-versa. People should probably read the other tutorial before reading this.